artifacts/centos/centos.go

```diff
@@ -126,6 +128,16 @@ func getAdditionalTags(version, variant, candidate string) (additionalTags []str
 	return
 }
 
+func getImageArchitecture(arch string) string {
+	if arch == "x86_64" {
+		return "amd64"
+	} else if arch == "aarch64" {
+		return "arm64"
+	}
+
+	return "unknown"
+}
+
```

Comment:
Two things:

1. Let's store this in a common place, so it can be reused from multiple places.
2. IMO a switch case looks better, maybe even a map to store the values.

Reply:
Done.